### ISSUE TITLE
NuGet 4.6 fails when verifying signed packages on Mono

### ISSUE THREAD

**Issue by marek-safar** [mono/mono#7413]
This is tracking issue for https://github.com/NuGet/Home/issues/6634

**Comment by marek-safar** [comment]
With the latest fixes, I get the following result for the test

```
Verifying TestPackage.AuthorSigned.1.0.0
/Users/marek/git/temp/nuget-test/TestPackage.AuthorSigned.1.0.0.nupkg

Signature Hash Algorithm: SHA256
Signature type: Unknown
WARNING: NU3027: The primary signature should be timestamped to enable long-term signature validity after the certificate has expired.
Verifying signature with author certificate:
Subject Name: CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US
SHA1 hash: F404000FB11E61F446529981C7059A76C061631E
Issued by: CN=DigiCert SHA2 Assured ID Code Signing CA, OU=www.digicert.com, O=DigiCert Inc, C=US
Valid from: 2/26/2018 1:00:00 AM to 1/27/2021 1:00:00 PM

NU3021: UntrustedRoot
NU3018: RevocationStatusUnknown
Finished with 2 errors and 1 warnings.

Could not verify package(s) signature.
```

**Comment by akoeplinger** [comment]
With the latest NuGet 4.6.0 from https://dist.nuget.org/win-x86-commandline/v4.6.0/nuget.exe I don't get verification errors anymore with latest Mono master:

```bash
$ mono nuget.exe verify -Signatures TestPackage.AuthorSigned.1.0.0.nupkg

Verifying TestPackage.AuthorSigned.1.0.0
/Users/alexander/dev/mono/TestPackage.AuthorSigned.1.0.0.nupkg

Signature Hash Algorithm: SHA256
Signature type: Unknown
WARNING: NU3027: The primary signature should be timestamped to enable long-term signature validity after the certificate has expired.
Verifying signature with author certificate:
Subject Name: CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US
SHA1 hash: F404000FB11E61F446529981C7059A76C061631E
SHA256 hash: 3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE
Issued by: CN=DigiCert SHA2 Assured ID Code Signing CA, OU=www.digicert.com, O=DigiCert Inc, C=US
Valid from: 2/26/2018 1:00:00 AM to 1/27/2021 1:00:00 PM

WARNING: NU3018: RevocationStatusUnknown
Finished with 0 errors and 2 warnings.

Successfully verified package(s).
```

On Windows it doesn't print the timestamp and revocation warnings though so I'm currently looking into whether that's another issue.

**Comment by marek-safar** [comment]
@akoeplinger is it the same version @alexischr is bumping the dependency to?

**Comment by akoeplinger** [comment]
@marek-safar yes.

**Comment by marek-safar** [comment]
This is now blocked by https://github.com/NuGet/Home/issues/6652

**Comment by marek-safar** [comment]
@akoeplinger have you done the check so we can close the issue?
